xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -543,6 +539,12 @@ void CGUIWindowPVRGuide::GetViewTimelineItems(CFileItemList &items)
       RefreshTimelineItems();
     }
 
+    if (m_newTimeline != nullptr)
+    {
+      m_cachedTimeline = m_newTimeline;
+      m_newTimeline.reset();
```

Comment:
As discussed on IRC. Don't reset it. Use std::move to move it to m_cachedTimeline.